mx_nuget.package/mx_buildpackage.ps1

```diff
@@ -0,0 +1,7 @@
+Param(
+    [Parameter(Mandatory=$true)]
+    [string]$version
+)
+
+.\nuget.exe Pack .\mx_nuget.package\MendixBinaries.nuspec -Version $version -NoPackageAnalysis
```

Comment:
This will not work anymore since you are adding `mx_nuget.package` to the path while you're already in that folder.